cli.js

```diff
@@ -17,6 +17,7 @@ var cli = meow({
 		'',
 		'Options',
 		'  --json     Output the result as JSON',
+		'  --bytes    Output the result in megabytes per second',
 		'  --verbose  Output more detailed information'
 	]
 });
```

Comment:
Got it Thanks @sindresorhus All the changes suggested are incorporated.Is it good to go for merge?